### ISSUE TITLE
When use a 'map-get' to the part of CSS selector, quotes are output. 

### ISSUE THREAD

**Issue by kyaido** [sass/libsass#1940]
Is this the correct behavior?
Sample code is below.

**libsass**
http://www.sassmeister.com/gist/229aae6859bf4d199843

**ruby-sass** (expect)
http://www.sassmeister.com/gist/ccc70bd58d0970ff61a6

**Comment by xzyfer** [comment]
@kyaido thanks for the report. I've confirmed this is an issue. For now you can work around this by assigning the function result to a variable first.

```sass
$foo :map-get($prefix, bar);
#{$foo} .baz {
  color: red;
}
```

Or using `unquote`.

```sass
#{unquote(map-get($prefix, bar))} .baz {
  color: red;
}
```

**Comment by mgreter** [comment]
[Cannot reproduce with 3.3.3 and 3.3.4] [1]?

[1]: http://libsass.ocbnet.ch/srcmap/#Ly8gLS0tLQovLyBsaWJzYXNzICh2My4zLjIpCi8vIC0tLS0KCiRwcmVmaXg6KAoJZm9vOiBmb28sCgliYXI6ICNiYXIKKTsKCiN7bWFwLWdldCgkcHJlZml4LCBmb28pfSB7CiAgY29sb3I6IHJlZDsKfQoje21hcC1nZXQoJHByZWZpeCwgYmFyKX0gewogIGNvbG9yOiByZWQ7Cn0KCiN7bWFwLWdldCgkcHJlZml4LCBmb28pfSAuYmF6IHsKICBjb2xvcjogcmVkOwp9CiN7bWFwLWdldCgkcHJlZml4LCBiYXIpfSAuYmF6IHsKICBjb2xvcjogcmVkOwp9

**Comment by xzyfer** [comment]
Confirmed this is fixed in 3.3.4.
